q2_moshpit/kraken2/tests/test_classification.py

```diff
@@ -321,5 +321,121 @@ def test_classify_kraken_action(self, p1):
         p1.assert_called_with(ANY, exp_args)
 
 
+class TestKraken2ClassifyContigs(unittest.TestCase):
```

Comment:
To keep things consistent here, maybe you could either split the tests above into respective classes for reads and  MAGs or just include the tests below into the class that already contains the other ones? The way it is now makes the contig classification seem as if it was somehow special compared to the other ones...

Reply:
Hey @misialq, I was going to add tests for reads and mags that prove functionality as I have for the contigs, but there wasn't enough time for this release. I can go ahead and spilt each sequence type into its own test class, you're right that that'll be more consistent.